index.html

```diff
@@ -2795,18 +2795,18 @@ <h2>
         </h2>
         <pre class="idl">
           dictionary AddressErrorFields {
-            DOMString addressLineError;
-            DOMString cityError;
-            DOMString countryError;
-            DOMString dependentLocalityError;
-            DOMString languageCodeError;
-            DOMString organizationError;
-            DOMString phoneError;
-            DOMString postalCodeError;
-            DOMString recipientError;
-            DOMString regionError;
-            DOMString regionCodeError;
-            DOMString sortingCodeError;
+            DOMString addressLine;
```

Comment:
Unimportant nit: the order here is different than for AddressInit, which is itself different from PaymentAddress. *shrug*

Reply:
Gonna go through the spec after all these parts land and put things in order. Right now just want to get things tested and implemented.